site/docs/patterns/dashboards/analytical-dashboard.mdx

```diff
@@ -0,0 +1,336 @@
+
+The grid you apply on a dashboard should reflect intent for experience and how you expect the application to behave. It’s the designers’ responsibility to decide which regions should align to the grid:
+
+1. Partially aligned
+2. Fully aligned
+
+For the purpose of this pattern, the following examples use a desktop screen size of 1440 x 1024px and a large, eight-column grid in medium density. The gutters and margins for Salt layout grids are `--salt-spacing-300` across all densities.
+
+### Partially aligned
+
+The main body of the dashboard aligns to a layout grid and combines with surrounding fixed-width panels.
+
+If your dashboard is responsive, the main body region will dynamically resize at different breakpoints. The fixed panel remains the same when the user resizes the viewport, but it can be set to the width of your choice based on desired outcome. You should consider the different content types, density, and target screen size when making your decision.
+
+<ImageSwitcher
+  images={[
+    {
+      src: "/img/patterns/dashboards/dashboard_partially_aligned.png",
+      alt: "Partially aligned dashboard without layout grid in main body region",
+    },
+    {
+      src: "/img/patterns/dashboards/dashboard_partially_aligned_layout_grid.png",
+      alt: "Partially aligned dashboard with layout grid in main body region",
+    },
+  ]}
+  label="Layout grid"
+/>
+
+#### Example of a responsive main body
+
+<Diagram
+  src="/img/patterns/dashboards/responsive_main_body_1.png"
+  alt="Example showing a responsive main body"
+  border
+/>
+
+<Diagram
+  src="/img/patterns/dashboards/responsive_main_body_2.png"
+  alt="Example showing a responsive main body"
+  border
+/>
+
+<Diagram
+  src="/img/patterns/dashboards/responsive_main_body_3.png"
+  alt="Example showing a responsive main body"
+  border
+/>
+
+### Fully aligned
+
+The main body of the dashboard and surrounding panels all align to a layout grid.
+
+If your experience is responsive, use the underlying layout grid to set a minimum width for your surrounding panels and adjust your content area based on your predefined breakpoints.
+
+If your experience has a fixed resolution (i.e., isn’t responsive), users can continue to use a layout grid to design consistent layouts.
+
+In the following example, the panel width aligns to the first two columns of the underlying layout grid.
+
+<ImageSwitcher
+  images={[
+    {
+      src: "/img/patterns/dashboards/dashboard_fully_aligned.png",
+      alt: "Fully aligned dashboard without layout grid",
+    },
+    {
+      src: "/img/patterns/dashboards/dashboard_fully_aligned_layout_grid.png",
+      alt: "Fully aligned dashboard with layout grid",
+    },
+  ]}
+  label="Layout grid"
+/>
+
+## Dashboard regions
+
+From a high-level overview, the main content regions of an analytical dashboard are:
+
+- Main body region
+- Header region
+- Fixed panels
+
+<ImageSwitcher
```

Comment:
"Fix panel" words jumped between 2 versions. Also, i don't think using a switch toggle "Visual hierarchy breakdown" is accurately describing the two

Reply:
I've suggested using "Main body region breakdown" cc @bhoppers2008